scst-store/multicluster-setup.hbs.md

```diff
@@ -64,24 +64,13 @@ data:
 EOF
 ```
 
-### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate from the View cluster
+### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate data from the View cluster
 
 With your kubectl targeted at the View cluster, you can view AMR CloudEvent Handler's TLS
 CA certificate. To copy the CA certificate into a `ceh_ca.yaml` file:
```

Comment:
Since you deleted writing into `ceh_ca.yaml`, does this section still need to be there?:
```
To copy the CA certificate into a `ceh_ca.yaml` file
```

Reply:
Good catch